great_expectations/expectations/expectation.py

```diff
@@ -2608,6 +2609,34 @@ def get_validation_dependencies(
                         metric_value_kwargs=metric_kwargs["metric_value_kwargs"],
                     ),
                 )
+        if isinstance(execution_engine, SparkDFExecutionEngine):
```

Comment:
Are these sections the same? If so maybe you could change the if to check either SQL or Spark e.g. `if isinstance(execution_engine, (SqlAlchemyExecutionEngine, SparkDFExecutionEngine)):`

Reply:
great catch. Thank you :)